app.js

```diff
@@ -2,22 +2,25 @@ const checkLocalStorageData = () => {
   let data = JSON.parse(localStorage.getItem("data")) || [];
   let taskDone = JSON.parse(localStorage.getItem("doneData")) || [];
 
-  if (data.length === 0 && taskDone.length === 0) {
-    post.style.display = "none";
-    doneTaskdiv.style.display = "none";
-  } else if (data.length === 0) {
-    post.style.display = "none";
-    doneTaskdiv.style.display = "block"; 
-  } else if (taskDone.length === 0) {
-    doneTaskdiv.style.display = "none";
-    post.style.display = "block";
-  } else {
-    post.style.display = "block";
-    doneTaskdiv.style.display = "block";
-  }
+  post.style.display =
```

Comment:
post.style.display = data.length > 0 ? "block" : "none";
doneTaskdiv.style.display = taskDone.length > 0 ? "block" : "none";
Replace the whole if else condition with these two lines It will work fine